cypress-tests/cypress/e2e/ConnectorUtils/Adyen.js

```diff
@@ -1,114 +1,120 @@
+      refundStatus: "pending",
+      refundSyncStatus: "pending",
+      mandate_type: {
+        single_use: {
+          amount: 8000,
+          currency: "USD",
+        },
+      },
     },
-    "MandateMultiUseNo3DS": {
-        "card": successfulNo3DSCardDetails,
-        "currency": "USD",
-        "paymentSuccessfulStatus": "succeeded",
-        "paymentSyncStatus": "succeeded",
-        "refundStatus": "pending",
-        "refundSyncStatus": "pending",
-        "mandate_type": {
-            "multi_use": {
-                "amount": 8000,
-                "currency": "USD"
-            }
-        }
+    MandateMultiUseNo3DS: {
+      card: successfulNo3DSCardDetails,
+      currency: "USD",
+      paymentSuccessfulStatus: "succeeded",
+      paymentSyncStatus: "succeeded",
+      refundStatus: "pending",
+      refundSyncStatus: "pending",
+      mandate_type: {
+        multi_use: {
+          amount: 8000,
+          currency: "USD",
+        },
+      },
     },
-    "MandateMultiUse3DS": {
-        "card": successfulThreeDSTestCardDetails,
-        "currency": "USD",
-        "paymentSuccessfulStatus": "requires_customer_action",
-        "paymentSyncStatus": "processing",
-        "refundStatus": "pending",
-        "refundSyncStatus": "pending",
-        "mandate_type": {
-            "multi_use": {
-                "amount": 8000,
-                "currency": "USD"
-            }
-        }
+    MandateMultiUse3DS: {
+      card: successfulThreeDSTestCardDetails,
+      currency: "USD",
+      paymentSuccessfulStatus: "requires_customer_action",
+      paymentSyncStatus: "processing",
+      refundStatus: "pending",
+      refundSyncStatus: "pending",
+      mandate_type: {
+        multi_use: {
+          amount: 8000,
+          currency: "USD",
+        },
+      },
     },
-    "SaveCardUseNo3DS": {
-        "card": successfulNo3DSCardDetails,
-        "currency": "USD",
-        "paymentSuccessfulStatus": "succeeded",
-        "paymentSyncStatus": "succeeded",
-        "refundStatus": "succeeded",
-        "refundSyncStatus": "succeeded",
-        "setup_future_usage": "on_session",
-        "customer_acceptance": {
-            "acceptance_type": "offline",
-            "accepted_at": "1963-05-03T04:07:52.723Z",
-            "online": {
-                "ip_address": "127.0.0.1",
-                "user_agent": "amet irure esse"
-            }
+    SaveCardUseNo3DS: {
+      card: successfulNo3DSCardDetails,
+      currency: "USD",
+      paymentSuccessfulStatus: "succeeded",
+      paymentSyncStatus: "succeeded",
+      refundStatus: "succeeded",
+      refundSyncStatus: "succeeded",
+      setup_future_usage: "on_session",
+      customer_acceptance: {
+        acceptance_type: "offline",
+        accepted_at: "1963-05-03T04:07:52.723Z",
+        online: {
+          ip_address: "127.0.0.1",
+          user_agent: "amet irure esse",
         },
+      },
+    },
+  },
+  bank_transfer_pm: {
```

Comment:
Please follow the PascalCase format that is being used at all the places

Reply:
Why? If I change `bank_transfer_pm` to `bankTransferPm`, I would also have to change `card_pm` to `cardPm` and make corresponding changes everywhere.